Src/Web/Web.Shared.Net/Extensibility/Implementation/PostSamplingTelemetryProcessor.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace Microsoft.ApplicationInsights.Web.Extensibility.Implementation
+{
+    using System;
+
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.Common;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing;
+    using Microsoft.ApplicationInsights.Web.Implementation;
+
+    /// <summary>
+    /// [This feature is still being evaluated and not recommended for end users.]
```

Comment:
put this into `<remark>` section, not on the top of description

Reply:
done